spec/features/consumer/shopping/checkout_spec.rb

```diff
@@ -39,7 +30,6 @@
 
     distributor.shipping_methods << free_shipping
     distributor.shipping_methods << shipping_with_fee
-    distributor.shipping_methods << tagged_shipping
```

Comment:
Is this tagged shipping behavior covered elsewhere? Why was it necessary to remove it from the checkout spec?

Reply:
I have split the checkout_spec in two files and then I deleted some things that are not needed in each of these two files. This line is still needed in checkout_guest_spec, but not in this one any more.